packages/components/src/components/base/models/model.ts

```diff
@@ -809,10 +811,10 @@ export class QueryGridModel extends Record({
                 return filterList.concat(this.getDetailFilters()).toList();
 
             }
-            return filterList.concat(this.baseFilters.concat(this.queryInfo.getFilters(this.view)).concat(this.filterArray)).toList();
+            return filterList.concat(baseFilters.concat(this.queryInfo.getFilters(this.view)).concat(this.filterArray)).toList();
```

Comment:
Seems possible that `this.filterArray` could also be undefined.  Does the concat of undefined do the right thing?

Reply:
Good catch. I pushed an update to do the same check on that object.